docs/how-to-guides/environment-setup-and-maintenance/isis-data-area.md

```diff
@@ -0,0 +1,185 @@
+    Downloading *all* takes over 2TB of space!  See [Partial Downloads](#partial-downloads) if you want a smaller subset, or try the [ISIS SPICE Web Service](#isis-spice-web-service)
+
+### Partial Downloads
+
+#### The Base Data Area
+
+Some data (like DEMs and Leap Second Kernels) are shared between multiple missions; this is the base data area.
+
+If you work with camera models (using `cam2map`, `campt`, `qview`, etc.), or are using a legacy version of ISIS, we recommend downloading the base data area:
+
+=== "Current"
+
+    ```sh
+    downloadIsisData base $ISISDATA
+    ```
+
+=== "Legacy"
+
+    ```sh
+    # For Isis versions before 4.1.0
+    downloadIsisData legacybase $ISISDATA
+    ```
+
+</br>
+
+#### Mission Specific Data Areas
+
+If you are only working with a few missions, save space by downloading just those data areas (find your download command on the following list).
+
+??? abstract "Mission Names & Download Commands"
+
+    For versions of ISIS prior to ISIS 4.1.0, please use the `--legacy` flag.
+
+    | Mission | Command |
+    | ------ | ------ |
+    | Apollo 15 | `downloadIsisData apollo15 $ISISDATA` |
+    | Apollo 16 | `downloadIsisData apollo16 $ISISDATA` |
+    | Apollo 17 | `downloadIsisData apollo17 $ISISDATA` |
+    | Cassini | `downloadIsisData cassini $ISISDATA` | 
+    | Chandrayaan 1 | `downloadIsisData chandrayaan1 $ISISDATA` |
+    | Clementine 1 | `downloadIsisData clementine1 $ISISDATA` |
+    | Dawn | `downloadIsisData dawn $ISISDATA` |
+    | ExoMars | `downloadIsisData tgo $ISISDATA` |
+    | Galileo | `downloadIsisData galileo $ISISDATA` | 
+    | Hayabusa 2 | `downloadIsisData hayabusa2 $ISISDATA` |
+    | Juno | `downloadIsisData juno $ISISDATA` |
+    | Kaguya | `downloadIsisData kaguya $ISISDATA` |
+    | Lunar Orbiter | `downloadIsisData lo $ISISDATA` |
+    | Lunar Reconnaissance Orbiter | `downloadIsisData lro $ISISDATA` |
+    | Mars Exploration Rover  | `downloadIsisData mer $ISISDATA` |
+    | Mariner10  | `downloadIsisData mariner10 $ISISDATA` |
+    | Messenger | `downloadIsisData messenger $ISISDATA` |
+    | Mars Express  | `downloadIsisData mex $ISISDATA` |
+    | Mars Global Surveyor  | `downloadIsisData mgs $ISISDATA` |
+    | Mars Reconnaissance Orbiter  | `downloadIsisData mro $ISISDATA` |
+    | Mars Science Laboratory  | `downloadIsisData msl $ISISDATA` |
+    | Mars Odyssey  | `downloadIsisData odyssey $ISISDATA` |
+    | Near  | `downloadIsisData near $ISISDATA` |
+    | New Horizons  | `downloadIsisData newhorizons $ISISDATA` |
+    | OSIRIS-REx  | `downloadIsisData osirisrex $ISISDATA` |
+    | Rolo  | `downloadIsisData rolo $ISISDATA` |
+    | Rosetta  | `downloadIsisData rosetta $ISISDATA` |
+    | Smart1  | `downloadIsisData smart1 $ISISDATA` |
+    | Viking 1 | `downloadIsisData viking1 $ISISDATA` |
+    | Viking 2 | `downloadIsisData viking2 $ISISDATA` |
+    | Voyager 1 | `downloadIsisData voyager1 $ISISDATA` |
+    | Voyager 2 | `downloadIsisData voyager2 $ISISDATA` |
+
```

Comment:
Im worried this will get outdated fast, maybe instead link to the rclone config?

Reply:
I added a link to rclone.conf, but since this is not a developer doc, I recommend leaving this table here for easier reference.